tests/test_other_formats_analysis.py

```diff
@@ -0,0 +1,61 @@
+import hashlib
+import json
+import pytest
+
+from udata_hydra.analysis.csv import analyse_csv
+
+pytestmark = pytest.mark.asyncio
+
+
+async def test_csvgz_analysis(setup_catalog, rmock, db, fake_check, produce_mock):
```

Comment:
You could use `pytest.mark.parametrize` on filename and expected count between the tests?
[Example use](https://github.com/opendatateam/udata-metrics/blob/edfd98e152aa7e28b381520cdd936bfccf6563bd/tests/test_metrics.py#L15).